docs/product-getting-started-guide/src/main/asciidoc/new-project-proc.adoc

```diff
@@ -11,12 +11,12 @@ For more information on Red Hat Technology Preview support, see https://access.r
 
 .Procedure
 
-Navigate to the Mortgage_Process sample project to view the pre-defined assets.
+Navigate to the Mortgage Process sample project to view the pre-defined assets.
```

Comment:
Remove this line, and consolidate the first two steps into one: "1. Log in to {CENTRAL} and click..." I'd search all cases of where logging in is its own step and consolidate. Need to streamline.